test/builders/remote/file.js

```diff
@@ -11,17 +13,32 @@ export default class RemoteFileBuilder extends RemoteBaseBuilder {
   constructor (cozy) {
     super(cozy)
 
-    this.data = `Content of remote file ${fileNumber}`
+    this._data = `Content of remote file ${fileNumber}`
 
     Object.assign(this.options, {
       name: `remote-file-${fileNumber++}`,
       contentType: undefined
     })
   }
 
+  contentType (contentType) {
+    this.options.contentType = contentType
+    return this
+  }
+
+  data (data) {
+    this._data = data
+    return this
+  }
+
+  dataFromFile (path) {
+    this._data = fs.readFileSync(path)
```

Comment:
Reminder for myself:
- Make sure `cozy-client-js` effectively picks up `node-fetch` as its `fetch` implementation
- I believed `node-fetch` supported buffers for the body... But `cozy-client-js` detects and uses the `.buffer` property. So maybe take a look at the typechecks in `node-fetch`?
- `node-fetch` is supposed to play well with streams, but `cozy-client-js` doesn't seem to, even on the server... PR?